src/main/java/seedu/address/logic/commands/room/InitRoomCommand.java

```diff
@@ -16,6 +16,8 @@
     public static final String MESSAGE_ZERO_CANNOT_BE_AN_INPUT = "Please input a positive value";
     public static final String MESSAGE_NEGATIVE_VALUES_CANNOT_BE_INPUT = "Please check your value! "
             + "You have input a negative value!";
+    public static final String MESSAGE_LARGE_NUMBER_OF_ROOMS_INPUT = "Please reduce the number of rooms input."
+            + "maximum number of rooms are 5000";
```

Comment:
maximum number...is 5000, just small gramma

Reply:
I was thinking if we should reduce the maximum number of rooms further since our target audience is small boutique hotels?